mapApp/urls.py

```diff
@@ -70,5 +70,5 @@
 	url(r'^thefts_tiny/?$', views.TinyTheftList.as_view(), name='tiny-thefts-list'),
 	url(r'^theft_xhr/?$', views.XHRTheftInfo.as_view(), name='xhr-thefts-detail'),
 	url(r'^newInfrastructures_tiny/?$', views.TinyNewInfrastructureList.as_view(), name='tiny-newInfrastructures-list'),
-	url(r'^newInfrastructures_xhr/?$', views.XHRNewInfrastructureInfo.as_view(), name='xhr-newInfrastructures-detail'),
-], allowed=['json'])
\ No newline at end of file
+	url(r'^newInfrastructure_xhr/?$', views.XHRNewInfrastructureInfo.as_view(), name='xhr-newInfrastructures-detail'),

```

Comment:
There is a very small chance that someone, somewhere is consuming our API. If we change this URL pattern it could break something. It would be safer to leave the original pattern in place, even if we're not consuming it internally, and add the additional pattern. It is a little redundant, but generally speaking, it is good practice to avoid breaking changes to your API. Like I said, probably not a big deal for this project, but something to be aware of in the future.

Reply:
that's a great point! keeping both.